WordPress/gutenberg
# Query Loop: Add filter by year

## What?

Closes https://github.com/WordPress/gutenberg/issues/58401

## Why?

This is enhancement to query loop block, which will have filter by year option in block setting.

## Testing Instructions

- Add query loop block
- Go to block setting, open filters
- Set year, and notice posts list.

## Screenshots or screencast
